### ISSUE TITLE
Doesn't work on 1.5.0-beta1 (or 1.6)

### ISSUE THREAD

**Issue by PallHaraldsson** [JuliaDatabases/ODBC.jl#297]
First got error for DecFP, but after fixing that:

```
(@v1.5) pkg> st DecFP
Status `~/.julia/environments/v1.5/Project.toml`
  [55939f99] DecFP v0.4.10 `https://github.com/JuliaMath/DecFP.jl.git#master`

julia> @time using ODBC
[ Info: Precompiling ODBC [be6f12e9-ca4f-5eb2-a339-a4f995cc0291]
ERROR: InitError: could not load symbol "SQLAllocHandle":
/home/pharaldsson_sym/julia-1.5.0-beta1/bin/julia: undefined symbol: SQLAllocHandle
Stacktrace:
 [1] SQLAllocHandle at /home/pharaldsson_sym/.julia/packages/ODBC/mRJax/src/API.jl:106 [inlined]
 [2] ODBCAllocHandle at /home/pharaldsson_sym/.julia/packages/ODBC/mRJax/src/ODBC.jl:166 [inlined]
 [3] __init__() at /home/pharaldsson_sym/.julia/packages/ODBC/mRJax/src/ODBC.jl:241
 [4] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
 [5] _require_from_serialized(::String) at ./loading.jl:749
 [6] _require(::Base.PkgId) at ./loading.jl:1040
 [7] require(::Base.PkgId) at ./loading.jl:928
 [8] require(::Module, ::Symbol) at ./loading.jl:923
 [9] top-level scope at timing.jl:174
during initialization of module ODBC
```

**Comment by quinnj** [comment]
You're on an old version of the package; you can update by doing `] up ODBC` and ensure you get `1.0.X`

**Comment by PallHaraldsson** [comment]
Yes, thought some but no only (@v1.6) pkg> add ODBC#v1.0.1
worked as I got downgraded, then tying to upgrade back:

```
(@v1.6) pkg> add SpecialFunctions#v0.10.3
    Cloning git-repo `https://github.com/JuliaMath/SpecialFunctions.jl.git`
  Resolving package versions...
ERROR: Unsatisfiable requirements detected for package SpecialFunctions [276daf66]:
 SpecialFunctions [276daf66] log:
 ├─possible versions are: 0.10.3 or uninstalled
 ├─SpecialFunctions [276daf66] is fixed to version 0.10.3
 └─found to have no compatible versions left with DecFP [55939f99] 
   └─DecFP [55939f99] log:
     ├─possible versions are: 0.4.5-0.4.10 or uninstalled
     └─restricted to versions 0.4.10-0.4 by ODBC [be6f12e9], leaving only versions 0.4.10
       └─ODBC [be6f12e9] log:
         ├─possible versions are: 1.0.1 or uninstalled
         └─ODBC [be6f12e9] is fixed to version 1.0.1
```
